docs/source/technical_tutorials/working_with_r/vector data visualization.ipynb

```diff
@@ -0,0 +1,533 @@
+{
```

Comment:
<p>The search above is for a GEDI 2A granule, but the visualization is for GEDI 4A granule.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>